virttest/data_dir.py

```diff
@@ -271,7 +258,7 @@ def clean_tmp_files():
 
 
 if __name__ == '__main__':
-    print("root dir:         " + ROOT_DIR)
+    print("root dir:         " + get_root_dir())
```

Comment:
given the root_dir lost most of it's significance we might just remove it from this listing, what do you think?

Reply:
It is still useful to print it though and this doesn't harm in any way or really increase complexity.